code/01_combine_submission.R

```diff
@@ -0,0 +1,49 @@
+################################################################################
+#                                                                                    
+#   Chronic Pain Publication                                                         
+#   Script: 01_combine_submission                                                     
+#                                                                                    
+#   Written for: R Studio Server                                                     
+#   R Version: 3.6.1
+#
+#   Packages required: here(1.0.1), purrr(0.3.4)
+#                      
+#                                                                                   
+#   Description: This script uses map_dfr to run the read_submisson function  
+#                and combine 14 quarterly submission files                                
+#                                                                                   
+################################################################################
+
+### 1 - Get file names and set path ----
+
+# Get list of files
+files <- walk(list.files(here("data", "submissions"), full.names = TRUE), source)
+
+# Path for reading data from the 14 submissions in the data/submissions folder
+#path_readsubmissions = paste0(here("data", "submissions", files, ".xlsx"))
+#path_readsubmissions = str_glue(here("data", "submissions", {files} ".xlsx"))
+
+# Set path to use in the read_submission function
+path_readsubmissions = here("data", "submissions", files, ".xlsx")
+
+
+### 2 - Use map() to run the read_submission function
+current_quarter <- map_dfr(files, read_submission)
+
+
+### 3 - Write file with current_quarter_end date included in file name  ----
+
+# NEED TO decide if writing current_quarter file is needed longer-term? Or would 
+# current_quarter be created in R and then added to the bottom of the ALL DATA 
+# file without saving current_quarter first?
```

Comment:
Writing and reading uses a lot of compute. I think the preferable option would be to keep everything within R and attempt to do all the necessary processing. Any outputs should ideally be the final outputs for the publication.